Public/Src/Engine/UnitTests/Scheduler.IntegrationTest/IncrementalSchedulingTests/GraphChangesTests.cs

```diff
@@ -1264,7 +1263,42 @@ public void GAISSCannotBeReusedFromEngineStateWhenGraphChange()
             result = RunScheduler(schedulerState: result.SchedulerState).AssertScheduled(q.PipId);
 
             AssertVerboseEventLogged(EventId.IncrementalSchedulingReuseState);
-            AssertLogContains(false, "Attempt to reuse existing incremental scheduling state: " + ReuseKind.ChangedGraph);
+            AssertLogContains(false, "Attempt to reuse existing incremental scheduling state from engine state: " + ReuseFromEngineStateKind.ChangedGraph);

```

Comment:
>Attempt to reuse existing incremental scheduling state from engine state [](start = 38, length = 72)

Consider replacing this with the DX code so the text is flexible if we ever change message strings again

Reply:
This test tests the reason of ISS' reusability. We verify the change kind.